sax/sax-tests.ts

```diff
@@ -17,7 +17,9 @@ parser.onerror = function(e: Error) {
 parser.ontext = function(text: string) {
 };
 
-parser.onopentag = function(tag: sax.Tag) {
```

Comment:
I'd recommend adding a test that asserts that properties in `tag.attributes` have type `string` for `sax.Tag`, and then something that reads the `QualifiedName` properties for `sax.QualifiedTag`. For example:

```ts
parser.onopentag = function(tag: sax.Tag) {
  var attribValue: string = tag.attributes["foo"];
};

// if opt.xmlns === true
parser.onopentag = function(tag: sax.QualifiedTag) {
  var prefix: string = tag.prefix;
  var attribPrefix: string = tag.attributes["foo"].prefix;
  var attribValue: string = tag.attributes["foo"].value;
};
```